maxim/test_runs/sanitization_utils.py

```
@@ -159,6 +168,23 @@ def sanitize_data(
+                                not (isinstance(value, list))
+                                or not (all(isinstance(v, str) for v in value))
+                            )
+                        ):
+                            raise ValueError(
+                                f'File URL variable column "{key}" has a data entry which is not a string',
+                                {
+                                    "cause": json.dumps(
+                                        {"dataEntry": {key: value}}, indent=2
+                                    )
+                                },
+                            )
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)